infra/scripts/createlocalsettings.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+
+if [ ! -f "./app/local.settings.json" ]; then
+
+    output=$(azd env get-values)
+
+    # Initialize variables
+    AISearchEndPoint=""
+    OpenAIEndPoint=""
+
+    # Parse the output to get the endpoint URLs
+    while IFS= read -r line; do
+        if [[ $line == *"AZURE_AISEARCH_ENDPOINT"* ]]; then
+            AISearchEndPoint=$(echo "$line" | cut -d '=' -f 2 | tr -d '"')
+        fi
+        if [[ $line == *"AZURE_OPENAI_ENDPOINT"* ]]; then
+            OpenAIEndPoint=$(echo "$line" | cut -d '=' -f 2 | tr -d '"')
+        fi
+    done <<< "$output"
+
+    cat <<EOF > ./app/local.settings.json
+{
+    "IsEncrypted": "false",
+    "Values": {
+        "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+        "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated",
```

Comment:
```suggestion
        "FUNCTIONS_WORKER_RUNTIME": "python",
```

Must be set to python here in local.settings.json or your function app will not start locally